lib/src/realm_object.dart

```diff
@@ -327,7 +328,7 @@ class RealmObjectChanges<T extends RealmObject> {
   /// The property names that have changed.
   List<String> get properties {
     final propertyKeys = realmCore.getObjectChangesProperties(_handle);
-    return object._realm!.getPropertyNames(object.runtimeType, propertyKeys);
+    return object.realm.getPropertyNames(object.runtimeType, propertyKeys);
```

Comment:
Now we can access the `_realm`, no need to go through the `realm` getter hop

Reply:
Erhh.. do you prefer `_realm!.`over `realm`?

Reply:
I don't mind either. I wonder about performance, but maybe its negligible. Lets leave it like that